lib/team.rb

```diff
@@ -1,11 +1,11 @@
 class Team
-    attr_reader :team_id, :franchiseId, :teamName, :abbreviation,
+    attr_reader :team_id, :franchiseId, :teamname, :abbreviation,
                 :stadium
 
     def initialize(team_info)
         @team_id = team_info[:team_id].to_i
         @franchiseId = team_info[:franchiseId].to_i
-        @teamName = team_info[:teamName]
+        @teamname = team_info[:teamname]
```

Comment:
This shift will support the building of our offense methods. We need to access team names.